IntegrationWithPharmacies/EPrescriptionApi/Model/RegistrationInPharmacy.cs

```diff
@@ -15,14 +12,14 @@ public class RegistrationInPharmacy : Entity
 
         public RegistrationInPharmacy() : base() { }
 
-        public RegistrationInPharmacy(int id, int idPharmacy, String apiKeyPharmacy,String pharmacyName, String pharmacyTown) : base(id)
+        public RegistrationInPharmacy(int id, int idPharmacy, String apiKeyPharmacy, String pharmacyName, String pharmacyTown) : base(id)
         {
             PharmacyId = idPharmacy;
             ApiKey = apiKeyPharmacy;
             Name = pharmacyName;
             Town = pharmacyTown;
         }
-          public RegistrationInPharmacy(int idPharmacy, String apiKeyPharmacy,String pharmacyName, String pharmacyTown) 
+        public RegistrationInPharmacy(int idPharmacy, String apiKeyPharmacy, String pharmacyName, String pharmacyTown)
         {
             PharmacyId = idPharmacy;
```

Comment:
Change parameter names to match for example: PharmacyId = pharmacyId
Apply  this logic in all constructors